<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="mainFrame">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>160</width>
      <height>220</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QFrame" name="item0_frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>160</width>
       <height>44</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="item0_label">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>8</y>
        <width>150</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>6</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="item0_pbar">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>27</y>
        <width>150</width>
        <height>10</height>
       </rect>
      </property>
      <property name="value">
       <number>25</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="item1_frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>44</y>
       <width>160</width>
       <height>44</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="item1_label">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>8</y>
        <width>150</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>6</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="item1_pbar">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>27</y>
        <width>150</width>
        <height>10</height>
       </rect>
      </property>
      <property name="value">
       <number>25</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="item2_frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>88</y>
       <width>160</width>
       <height>44</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="item2_label">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>8</y>
        <width>150</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>6</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="item2_pbar">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>27</y>
        <width>150</width>
        <height>10</height>
       </rect>
      </property>
      <property name="value">
       <number>25</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="item3_frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>132</y>
       <width>160</width>
       <height>44</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="item3_label">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>8</y>
        <width>150</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>6</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="item3_pbar">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>27</y>
        <width>150</width>
        <height>10</height>
       </rect>
      </property>
      <property name="value">
       <number>25</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="item4_frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>176</y>
       <width>160</width>
       <height>44</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="item4_label">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>8</y>
        <width>150</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>6</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="item4_pbar">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>27</y>
        <width>150</width>
        <height>10</height>
       </rect>
      </property>
      <property name="value">
       <number>25</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="topBar_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QPushButton" name="topBar_button0">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="topBar_button3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>0</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="topBar_button2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>down</string>
     </property>
    </widget>
    <widget class="QPushButton" name="topBar_button1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>up</string>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>160</width>
      <height>220</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
